src/ConfigurationContext.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FunctionComponent, createContext, useContext } from "react"
+import { Platform } from "react-native"
+import env from "react-native-config"
+
+export interface Configuration {
+  displaySelfAssessment: boolean
+  displayReportAnIssue: boolean
+  appDownloadLink: string
+  authorityPrivacyPolicyUrl: string
+  authorityName: string
+  authorityAdviceUrl: string
+}
+
+const initialState = {
+  displaySelfAssessment: false,
+  displayReportAnIssue: false,
+  appDownloadLink: "",
+  authorityPrivacyPolicyUrl: "",
+  authorityName: "",
+  authorityAdviceUrl: "",
+}
+
+const ConfigurationContext = createContext<Configuration>(initialState)
+
+const DEFAULT_DOWNLOAD_LINK = ""
+const DEFAULT_AUTHORITY_PRIVACY_URL = ""
+const DEFAULT_AUTHORITY_NAME = ""
+const DEFAULT_AUTHORITY_ADVICE_URL = ""
```

Comment:
This is another benefit of controlling the environment here, we could:
- Display warnings on inconsistent configurations in a dev environment
- Have a safeguard for miss-configured builds, so no blowing up if something is missing while we fix it.

Reply:
Maybe we don't need a default since these configs should be vetted in advance but the paranoid in me likes this.

Reply:
I dig this a lot. It would also be nice to see some build time checking as well though, if we dont have a valid config file it would be nice to fail at compile time.